### PR TITLE
Ignore roots of groupfolders from autotagging

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: lib/Operation.php [hunk 1]
```diff
@@ -106,11 +106,11 @@ public function isTaggingPath(IStorage $storage, string $file): bool {
 			list($folder) = explode('/', $file, 2);
 			return $folder === 'files';
 		} else {
-			list($folder) = explode('/', $file, 2);
+			[$folder, $subPath] = explode('/', $file, 2);
```
Comment (nickvergessen) [inline lib/Operation.php]: There might not be a slash, so this needs more fixing

Comment (juliushaertl) [reply lib/Operation.php]: Pushed a commit to return early so we don't try to explode the file path if if just contains the base `__groupfolders` path

**Comment by nickvergessen** [comment]
What happened here, the PR is against stable16? :P

**Comment by juliushaertl** [comment]
Ah right, then I guess it needs some forward-porting. :see_no_evil:

**Comment by nickvergessen** [comment]
Rebased onto latest master. It actually was master based, that's why the unit tests against stable16 failed :D
So we can merge this once the tests ran

**Comment by nickvergessen** [comment]
Failing tests are due to #85 

So lets just merge this now

**Pull request merged**